src/hotspot/share/runtime/fieldDescriptor.cpp

```diff
@@ -44,7 +44,7 @@ Symbol* fieldDescriptor::generic_signature() const {
 
 bool fieldDescriptor::is_trusted_final() const {
   InstanceKlass* ik = field_holder();
-  return is_final() && (is_static() || ik->is_hidden() || ik->is_record());
+  return is_final() && (is_static() || ik->is_hidden() || ik->is_record() || ik->name()->equals("java/lang/StableValue") == 0);
```

Comment:
I think we should check if the field type is `StableValue` too.

We should define a new entry for `StableValue` in vmSymbols.hpp, and write something like
```
signature() == vmSymbols::java_lang_StableValue()
```
to make stablevalue typed fields trusted.

Reply:
We could add something like

```
  /* StableValues & Collections */                                                                                \
  template(java_lang_StableValue,                 "java/lang/StableValue")                                       \
```

in vmSymbols.hpp. But I am unsure how to define `java_lang_StableValue`. Could you shed some light on this?